cmd/kured/main.go

```diff
@@ -91,6 +94,12 @@ func main() {
 		PreRun: flagCheck,
 		Run:    root}
 
+	rootCmd.PersistentFlags().BoolVar(&forceReboot, "force-reboot", false,
+		"force a reboot even if the drain is still running (default false)")
+	rootCmd.PersistentFlags().IntVar(&drainGracePeriod, "drain-grace-period", -1,
+		"grace period of time for pods to wait for the node drain in seconds (default -1)")
+	rootCmd.PersistentFlags().DurationVar(&drainTimeout, "drain-timeout", time.Minute*30,
```

Comment:
Are we following any existing guidance here on setting 30 mins as the default drain timeout?

Reply:
Not exactly sure where that value came from. Looking at the code, the default should be zero (infinite).